From c20ed8df73a6fa1039c62863bf9b6e2ac165f483 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:07 -0400
Subject: [redhat] kabi: Adding symbol down_write_trylock (kernel/rwsem.c)

Message-id: <3b9cf8e495ff7020506c37f4ad07e7b94c129c73.1410510773.git.poros@redhat.com>
Patchwork-id: 95948
O-Subject: [PATCH RHEL7.1 067/233] kabi: Adding symbol down_write_trylock (kernel/rwsem.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Used to acquire semaphore for writing without waiting.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/down_write_trylock
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/down_write_trylock
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/down_write_trylock

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/down_write_trylock b/redhat/kabi/kabi-module/kabi_ppc64/down_write_trylock
new file mode 100644
index 0000000..77bfe83
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/down_write_trylock
@@ -0,0 +1,2 @@
+#1-
+0xbe77aebe down_write_trylock vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/down_write_trylock b/redhat/kabi/kabi-module/kabi_s390x/down_write_trylock
new file mode 100644
index 0000000..6b75bb7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/down_write_trylock
@@ -0,0 +1,2 @@
+#1-
+0x4850ec10 down_write_trylock vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/down_write_trylock b/redhat/kabi/kabi-module/kabi_x86_64/down_write_trylock
new file mode 100644
index 0000000..f1043d7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/down_write_trylock
@@ -0,0 +1,2 @@
+#1-
+0x35225ea3 down_write_trylock vmlinux EXPORT_SYMBOL
-- 
1.7.1